extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/error/NotEnoughMemoryFault.java

```diff
@@ -45,19 +47,27 @@ public NotEnoughMemoryFault(
   {
     super(
         CODE,
-        "Not enough memory (total = %,d; usable = %,d; server workers = %,d; server threads = %,d)",
+        "Not enough memory. Required alteast %,d bytes. (total = %,d bytes; usable = %,d bytes; server workers = %,d; server threads = %,d). Increase JVM memory with the -xmx option",
```

Comment:
Nit: `atleast` -> `at least`

Note that there seems to be a convention that all interpolated values be enclosed in angle brackets. Seems silly sometimes, bug some folks feel strongly about it.

Increasing JVM memory is not always possible. The other solution is to _decrease_ the number of workers. This is the problem with making suggestions: there are multiple ways to solve the problem.